mmdeploy/backend/ascend/utils.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+import os.path as osp
+
+from mmdeploy.utils import get_root_logger
+
+
+def update_sdk_pipeline(work_dir: str):
```

Comment:
What's this function for, especially considering dynamic (h, w) is supported by `detection_ascend_dynamic_800x1344`?

Reply:
Ascend does not support dynamic shape for any shape.
We can export two shapes [800x1344] & [1344x800], which need a special way to pad the input. This function is used to update the pad mode to support such transform.